src/apps/aave-v2/helpers/aave-v2.lending.token-helper.ts

```diff
@@ -172,11 +172,9 @@ export class AaveV2LendingTokenHelper {
             statsItems,
           },
         };
-
         return token;
       }),
     );
-
```

Comment:
Remove unnecessary whitespace changes here and in `src/apps/aave-v2/helpers/aave-v2.claimable.contract-position-helper.ts`